src/dmd/iasm.d

```diff
@@ -4558,3 +4558,20 @@ AFTER_EMIT:
     //return asmstate.bReturnax;
     return s;
 }
+
+/**********************
+ * If c is a power of 2, return that power else -1.
+ */
+
+private int ispow2(uint c)
```

Comment:
D shouldn't be about duplicating code :(

Reply:
There's a lot of hackish code that the conversion to D will eliminate.